### DISCUSSION TITLE
How to add installed library into context menu?

### DISCUSSION THREAD

**Discussion by vadhavana** [leokhoa/laragon#407] [Q&A] [answered]
I have downloaded **filezilla** using laragon by adding download url into **packages.conf**, installed filezilla is not sowing into c**ontext menu**. Any idea about it?

**Comment by Pen-y-Fan** [comment, marked as answer]
Laragon uses **Proc file** for custom packages. See my blog on MailHog:

- [How to set up Laragon on a new Windows computer (part 5) - MailHog](https://pen-y-fan.github.io/2023/01/04/how-to-set-up-laragon-on-a-new-windows-computer-part-5/#autostart),

![Mailhog in Proc file](https://user-images.githubusercontent.com/40126936/218026565-8d37a415-3487-45a6-950d-6738b3091b96.png)

autostart is optional. This will launch the package every time Laragon is executed.

Swap the **MailHog.exe** with the FileZilla executable. The full path to the executable may be required too.
